targets/targets.json

```diff
@@ -8499,7 +8499,7 @@
     },
     "MCU_PSOC6": {
         "inherits": ["Target"],
-        "macros": ["MBED_MPU_CUSTOM", "LPTICKER_DELAY_TICKS=3"],
+        "macros": ["MBED_MPU_CUSTOM", "MBEDTLS_CONFIG_HW_SUPPORT", "LPTICKER_DELAY_TICKS=3"],
         "default_toolchain": "GCC_ARM",
         "supported_toolchains": ["ARM", "GCC_ARM", "IAR"],
         "core": "Cortex-M4F",
```

Comment:
If you meant to remove the TRNG hardware support from software, you may also need to edit the `device_has` `TRNG` entry below here in `targets.json`.

Reply:
yes, you are right.

Reply:
It'd be good to support TRNG if your hardware is capable.

Reply:
Yes, will expand TRNG support in the next release.